src/workloads/transactions/LLTMixed.yml

```diff
@@ -5,16 +5,15 @@ Description: |
 
 GlobalDefaults:
```

Comment:
By my reading this is: 
0. Queisce
1. Initial Load
2. Warmup
3. Second Load

Then repeating: quiesce, warmup, test. Is that correct? 
If it is I can't find the phase 5 warmup. 

Could we document this flow?

Reply:
Done below.